mk/cmd.mk

```diff
@@ -18,5 +18,11 @@ $(CMD): $(SRCS) $(CMD_DEPS) ; go build -o $(CMD)
 
 CLEANFILES += $(CMD)
 
+.PHONY: realall
```

Comment:
Just out of curiosity, what does realall implies ? Can a better word be used here ?

Reply:
`real`+_`<target name>`_ is the convention I have used to name the _real_ (i.e., defined) action for all the "hook-able" targets